onnxruntime/core/providers/openvino/openvino_graph.cc

```diff
@@ -326,7 +332,7 @@ size_t OpenVINOGraph::DeduceBatchSize(Ort::CustomOpApi ort, const OrtValue* inpu
 
 // Starts an asynchronous inference request for data in slice indexed by batch_slice_idx on
 // an Infer Request indexed by infer_req_idx
-void OpenVINOGraph::StartAsyncInference(Ort::CustomOpApi ort, const OrtValue* input_tensors[],
+void OpenVINOGraph::StartAsyncInference(Ort::CustomOpApi ort, std::vector<const OrtValue*> input_tensors,
```

Comment:
can you give some background on this and other api signature changes?
now we want to use a vector for inputs and outputs and pass the vector by value? 
why is this needed for enabling windows build?

Reply:
MSVC++ does not support variable length array and was throwing build errors, so we had to use vector which passed windows build